src/lmic/config.h

```diff
@@ -69,7 +69,21 @@
 // communicating with the radio.
 // The standard range is 125kHz-8MHz, but some boards can go faster.
 #ifndef LMIC_SPI_FREQ
-#define LMIC_SPI_FREQ 1E6
+# ifdef ARDUINO_CATENA_4550
+#  define LMIC_SPI_FREQ 4000000	/* 4MHz */
```

Comment:
We could go to 10MHz.  I think we should move this to the wiring table -- if defaulted (zero) then use 1 MHz. But then in the 4551 we can ramp it up to 8 or 9 MHz (whatever is conveniently close to 10). Faster means less power wasted in the CPU waiting.